osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerMatchSettingsOverlay.cs

```diff
@@ -460,7 +461,7 @@ private void apply()
             });
         }
 
-        public class CreateOrUpdateButton : TriangleButton
+        public class CreateOrUpdateButton : RoundedButton
```

Comment:
This one isn't great when hovered either:

![osu_2022-11-24_22-31-52](https://user-images.githubusercontent.com/20418176/203865634-766d4e35-7296-43d5-af83-6e7a0819f8c5.jpg)

Reply:
There's a few like this. I guess I can adjust the colours in this PR for the ones I noticed. Luckily they are usually bad only when hovering.